web-ng/src/app/components/form-field-editor/form-field-editor.component.spec.ts

```diff
@@ -33,6 +37,9 @@ describe('FormFieldEditorComponent', () => {
     TestBed.configureTestingModule({
       declarations: [FormFieldEditorComponent],
       imports: [
+        AngularFireModule.initializeApp(environment.firebaseConfig),
```

Comment:
Why initialize Firebase here? For unit tests we wouldn't want to use a real database, but rather a mocked out instance of DataStore. Can that be done here?